server/src/main/java/org/eclipse/lsp/cobol/core/strategy/CobolErrorStrategy.java

```diff
@@ -43,28 +46,35 @@
   private static final String REPORT_UNWANTED_TOKEN = "reportUnwantedToken";
   private static final String REPORT_MISSING_TOKEN = "reportMissingToken";
   private static final String SPECIAL_TOKEN_HANDLING_FILEPATH = "SpecialTokenHandling.properties";
-  private static final LinkedHashMap<String, String> SPECIAL_TOKEN_HANDLING_MAP;
+  private static Map<String, String> specialTokenHandlingMap;
```

Comment:
Why it's not a final? Nothing was changed from that point of view.

Reply:
When it is final, it produces some error about initialization